src/components/AuthLayout/index.js

```diff
@@ -142,11 +134,7 @@ class AuthLayout extends Component {
         </GridItem>
         <GridItem>
           <Title headingLevel="h4" size="xl">
-            <Button
-              variant="link"
-              className={styles.inlineLink}
-              onClick={() => this.navigate('password')}
-            >
+            <Button variant="link" onClick={() => this.navigate('password')}>
```

Comment:
So it's smaller just because we're _not_ giving it the `inlineLink` style? Would it be better if the size was explicit rather than dependent on the surrounding context?

Reply:
i think if we are not using that, we should consider removing the `inlineLink` css definition